airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/BasicAcceptanceTests.java

```diff
@@ -394,32 +401,65 @@ void testScheduledSync() throws Exception {
     final UUID operationId = testHarness.createOperation().getOperationId();
     final AirbyteCatalog catalog = testHarness.discoverSourceSchema(sourceId);
 
-    final ConnectionSchedule connectionSchedule = new ConnectionSchedule().units(1L).timeUnit(MINUTES);
     final SyncMode syncMode = SyncMode.FULL_REFRESH;
     final DestinationSyncMode destinationSyncMode = DestinationSyncMode.OVERWRITE;
     catalog.getStreams().forEach(s -> s.getConfig().syncMode(syncMode).destinationSyncMode(destinationSyncMode));
 
-    final var connection =
-        testHarness.createConnection(TEST_CONNECTION, sourceId, destinationId, List.of(operationId), catalog, connectionSchedule);
-
-    // When a new connection is created, Airbyte might sync it immediately (before the sync interval).
-    // Then it will wait the sync interval.
-    // if the wait isn't long enough, failures say "Connection refused" because the assert kills the
-    // syncs in progress
-    List<io.airbyte.api.client.model.generated.JobWithAttemptsRead> jobs = new ArrayList<>();
-    while (jobs.size() < 2) {
-      final var listSyncJobsRequest = new io.airbyte.api.client.model.generated.JobListRequestBody().configTypes(List.of(JobConfigType.SYNC))
-          .configId(connection.getConnectionId().toString());
-      final var resp = apiClient.getJobsApi().listJobsFor(listSyncJobsRequest);
-      jobs = resp.getJobs();
+    final UUID connectionId =
+        testHarness.createConnection(TEST_CONNECTION, sourceId, destinationId, List.of(operationId), catalog, ConnectionScheduleType.BASIC,
+            BASIC_SCHEDULE_DATA).getConnectionId();
+
+    for (int i = 0; i < 10; i++) {
```

Comment:
This is basically just to impose a limit on how many times the test checks for the job to succeed, right? Might be worth a comment, or maybe worth adding some explicit logging in the case where we tried 10 times and the job was still waiting so that it's very clear in the case where this test fails